app/components/SearchBox.tsx

```diff
@@ -76,16 +75,6 @@ const SearchBox: React.FunctionComponent<Props> = ({
           </ButtonGroup>
         </div>
       </JsonSchemaForm>
-      {/* Hide searchField / searchValue JsonSchemaForm titles  */}
-      <style jsx global>
-        {`
-          .hide-title label.form-label {
-            display: none;
-          }
-        `}
```

Comment:
In addition to hiding the `<label>`s mentioned above (in SearchBox), this was actually polluting the global scope and hiding other `<label>`s associated with other `<input>`s.

It's no longer necessary, since we need to show labels.